README.md

```diff
@@ -71,3 +71,5 @@ Many people are currently supporting this app through <a href="https://github.co
 
 <a href="https://osmfoundation.org/"><img src=".github/images/logo_osmf.png" height="58"/></a><br/>
 The <a href="https://osmfoundation.org/">OpenStreetMap foundation</a> was funding the development of doing <a href="https://wiki.openstreetmap.org/wiki/Microgrants/Microgrants_2020/Proposal/Map_Maintenance_with_StreetComplete">map maintenance with StreetComplete</a> (~ 3 weeks) in their <a href="https://blog.openstreetmap.org/2020/07/01/osmf-microgrants-program-congratulations-to-selected-projects/">first round</a> of the <a href="https://wiki.osmfoundation.org/wiki/Microgrants">microgrant program</a> in 2020.
+
+<a rel="me" href="https://mastodon.social/@streetcomplete"></a>
```

Comment:
It seems to identify this social media account as official, while it is not.

If it is doing anything else - can you link relevant documentation?

Reply:
Here's the documentation about that:

https://docs.joinmastodon.org/user/profile/

The idea is to make the link in the profile verified. So it appears to the user to be "safe to click on" since it's the official link for the profile.

I do already operate the twitter profile, after talking to @westnordost about this via email to which terms he's comfortable with that.

I only want users to be sure that the link is safe if they click on it. :)

Reply:
Maybe @rugk can chime in here as well, as he recommended to create a mastodon account - how he feels about adding a verified link. :)

Reply:
Yeah. Its good and makes it look more official/supported by SC.

Reply:
> However, Mastodon can cross-reference the links you put on your profile to prove that you are the real owner of those > links.
> (...)
> If you put a link in your profile metadata, Mastodon checks if the linked page links back to your Mastodon profile. If so, you get a verification checkmark next to that link, since you are confirmed as the owner.

So it seems that it would authenticate you as an official profile of SC on Mastodon. It seems that

> It's not to verify the account, but the other way around.

is not an accurate description.

Reply:
https://docs.joinmastodon.org/user/profile/

> However, Mastodon can cross-reference the links you put on your profile to prove that you are the real owner of those links.
> (...)
> If you put a link in your profile metadata, Mastodon checks if the linked page links back to your Mastodon profile. If so, you get a verification checkmark next to that link, since you are confirmed as the owner.

So it seems that it would authenticate you as an official profile of SC on Mastodon. It seems that

> It's not to verify the account, but the other way around.

is not an accurate description.